tests/atom/clusterBuildTestsRun.sh

```diff
@@ -31,14 +75,15 @@ sudo docker run \
     quay.ceph.io/ceph-ci/ceph:"$CEPH_SHA" \
     quay.io/ceph/nvmeof:"$VERSION" \
     quay.io/ceph/nvmeof-cli:"$VERSION" \
-    None None None None None None 4 1 1 2 4 1024 2 2 200M 0 1 20 20 1 \
+    None None None None None None 1 1 4 1 1 2 4 1024 2 2 200M 0 1 20 10 1 \
     --stopNvmeofDaemon \
     --stopNvmeofSystemctl \
     --stopMonLeader \
     --rmNvmeofDaemon \
     --gitHubActionDeployment \
     --dontUseMTLS \
+    --skiplbGroupChangeTest \
```

Comment:
@barakda looks great. could we omit `skiplbGroupChangeTest`, sounds like a test we might want to run on each each PR cycle, wdyt?

Reply:
Thanks @baum , removed.